DesignSafe-CI/portal
# task/DES-2595: Add TRAM endpoint to account registration page

## Overview: ##
Allow users to request an account by calling the new TRAM invitation endpoint.

## PR Status: ##


## Related Jira tickets: ##

* [DES-2595](https://tacc-main.atlassian.net/browse/DES-2595)

## Summary of Changes: ##
Update the account registration page to add a form where users can request a project invite by entering their email and submitting a reCAPTCHA.

## Testing Steps: ##
1. Make sure that you have an updated `designsafe.env` file from Stache.
2. Go to https://designsafe.dev/account/register/ and fill in the form. Make sure you get a success message when you enter a valid email, and an error message when the captcha is left incomplete.

## UI Photos:
![image](https://github.com/DesignSafe-CI/portal/assets/12601812/8de50e91-5cb8-45ce-8216-2fc71252369c)


## Notes: ##
